queries/lua/injections.scm

```diff
@@ -40,7 +40,7 @@
 (function_call
   (dot_index_expression
     field: (identifier) @_method
-    (#eq? @_method "gmatch" "gsub"))
+    (#any-of? @_method "gmatch" "gsub"))
```

Comment:
Can we merge this with the pattern right above it? the `"find "match"` one. They are the same from what I can see

Reply:
Ah no, dismiss my comments